alert-common/src/main/java/com/synopsys/integration/alert/common/message/model/MessageContentKey.java

```diff
@@ -44,7 +43,32 @@ public static MessageContentKey from(final String topicName, final String topicV
         return new MessageContentKey(fullKey);
     }
 
+    public static MessageContentKey from(final String topicName, final String topicValue, final Set<LinkableItem> subTopics) {
+        if (null == subTopics || subTopics.isEmpty()) {
+            return from(topicName, topicValue);
+        }
+
+        final StringBuilder builder = new StringBuilder();
+        builder.append(topicName);
+        builder.append(SEPARATOR);
+        builder.append(topicValue);
+
+        for (final LinkableItem subTopic : subTopics) {
+            builder.append(SEPARATOR);
+            builder.append(subTopic.getName());
+            builder.append(SEPARATOR);
+            builder.append(subTopic.getValue());
```

Comment:
These can probably be collapsed as well.

Reply:
same as above